src/data_provider/src/packages/packageLinuxParserHelper.h

```diff
@@ -18,6 +18,7 @@
 #include "stringHelper.h"
```

Comment:
Add license comment block
```suggestion
/* Copyright (C) 2015-2021, Wazuh Inc.
 * All rights reserved.
 *
 * This program is free software; you can redistribute it
 * and/or modify it under the terms of the GNU General Public
 * License (version 2) as published by the FSF - Free Software
 * Foundation
 */
 
#include "stringHelper.h"
```